resources/function_help/json/to_dms

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "to_dms",
+  "type": "function",
+  "description": "Convert a coordinate to degree, minute, second.",
+  "arguments": [
+    {"arg":"coordinate","description":"A latitude or longitude value."},
+    {"arg":"axis","description":"The axis of the coordinate. Either 'x' or 'y'."},
+    {"arg":"precision", "description":"Number of decimals."},
+    {"arg":"formatting", "optional": true, "description":"Designates the formatting type. Acceptable values are NULL, 'aligned' or 'suffix'. The default is not aligned and without a suffix."}
```

Comment:
I was thinking more of something like https://github.com/qgis/QGIS/blob/fff07672410310eb185b2f079dcf62846fcca91c/resources/function_help/json/array_to_string#L7 (if I'm not wrong it formats the syntax)

Reply:
Oh ok